screenpy/actions/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .eventually import Eventually
 from .make_note import MakeNote
 from .pause import Pause
+from .quietly import Quietly, QuietlyAnswerable, QuietlyPerformable, QuietlyResolvable
```

Comment:
I don't think we want to import all the sub-`Quietly`s here, we don't intend the users to use them directly. If they need them for type hinting, i think it's OK to make them go to `screenpy.actions.quietly`.